lib/build.js

```diff
@@ -59,7 +59,7 @@ async function compileOnWindows (nodeVersion, targetArch) {
   return path.join(nodePath, 'out/Release/node.exe');
 }
 
-const { MAKE_JOB_COUNT = '1' } = process.env;
+const { MAKE_JOB_COUNT = require('os').cpus().length } = process.env;
```

Comment:
Is this cross-platform supported?

Reply:
It is a nodejs module so yes AFAIK, at least in docs it is not mentioned as platform related: https://nodejs.org/api/os.html#os_os_cpus

Reply:
Awesome! Thank you.